internal/identity/identitymanager.go

```diff
@@ -33,36 +34,44 @@ import (
 )
 
 type Manager interface {
-	ResolveInputIdentity(ctx context.Context, identity *fftypes.Identity) (err error)
-	ResolveSigningKey(ctx context.Context, inputKey string) (outputKey string, err error)
-	ResolveSigningKeyIdentity(ctx context.Context, signingKey string) (author string, err error)
-	ResolveLocalOrgDID(ctx context.Context) (localOrgDID string, err error)
-	GetLocalOrgKey(ctx context.Context) (string, error)
-	OrgDID(org *fftypes.Organization) string
-	GetLocalOrganization(ctx context.Context) (*fftypes.Organization, error)
+	ResolveInputSigningIdentity(ctx context.Context, namespace string, msgSignerRef *fftypes.SignerRef) (err error)
```

Comment:
Pretty much all newly rebuilt (as well as renamed) functions in this class, trying to address some previous sources of confusion, as well as introducing a better level of caching, and migrating to the new `Identity` object model.